install/kubernetes/cilium/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+You have successfully installed Cilium {{ title .Chart.Name }}
+
+Your release version is {{ .Chart.Version }}
```

Comment:
nit:

These are still regular sentences, so they should probably have punctuation right?

```suggestion
You have successfully installed Cilium {{ title .Chart.Name }}.

Your release version is {{ .Chart.Version }}.
```

Reply:
ok will address your comments.

Reply:
Done